packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -2448,6 +2454,10 @@ function flushPassiveEffectsImpl() {
   nestedPassiveUpdateCount =
     rootWithPendingPassiveEffects === null ? 0 : nestedPassiveUpdateCount + 1;
 
+  if (__DEV__) {
+    isFlushingPassiveEffects = false;
```

Comment:
Do we have a guarantee code in between wouldn't throw even if user code throws (unless there's a bug in React)?

Reply:
User code is try/catched.

Though I wonder if I should have moved this before `flushSyncCallbackQueue` since that's where we reset prev exceution context.